### PR TITLE
Fix incorrect rotation matrix after selecting nodes

### PR DESCRIPTION
**Pull request by philippotto**
Nodes that are created in arbitrary mode have an attached non-default rotation. Selecting that node will center their position and change apply their rotation. In orthogonal mode, this shouldn't happen as that rotation is used for bucket picking.

### URL of deployed dev instance (used for testing):
- https://fixchangingrotation.webknossos.xyz

### Steps to test:
- switch to flight mode with non-default angles (e.g., 45**3)
- create some nodes
- switch back to ortho mode and select nodes
- data should be rendered without problems

### Issues:
- second fix for #5288 

------
(Please delete unneeded items, merge only when none are left open)
- [x] Updated [changelog](../blob/master/CHANGELOG.unreleased.md#unreleased)

### CODE REVIEW

File: frontend/javascripts/oxalis/model/reducers/flycam_reducer.ts [hunk 1]
```diff
@@ -164,8 +164,11 @@ export function setDirectionReducer(state: OxalisState, direction: Vector3) {
     },
   });
 }
+
 export function setRotationReducer(state: OxalisState, rotation: Vector3) {
-  if (state.dataset != null) {
+  // Don't change rotation in orthogonal mode, because this would lead to incorrect
+  // buckets being selected.
```
Comment (philippotto) [inline frontend/javascripts/oxalis/model/reducers/flycam_reducer.ts:170]:
```suggestion
  // buckets being selected for rendering.
```

**Review by daniel-wer — APPROVED** [review]
Code LGTM :+1:

**Comment by daniel-wer** [comment]
I followed the testing instructions and the issue still occured :thinking: 

image

**Comment by philippotto** [comment]
> I followed the testing instructions and the issue still occured thinking

Argh, I think I had "prepared" nodes with rotations and after building the fix only ensured that selecting these nodes didn't cause rendering bugs.

> Edit: The issue actually already occurs when switching back to orthogonal, no node selection needed. Maybe, the additional condition prevents the rotation reset that's triggered when switching to orthogonal?

Yes, that was exactly it. Thanks for catching this :pray: :pray: :pray: 

Could you have another look for a final thumbs up?

**Review by daniel-wer — APPROVED** [review]
Nice, works well :+1:

**Pull request merged**